### PR TITLE
Add conversions for location and importedCase fields

### PR DESCRIPTION
**Pull request by khmoran**
Example output:

```
{
    "demographics": {
      "age": {
        "range": {
          "min": 53.0,
          "max": 53.0
        }
      },
      "sex": "Male"
    },
    "events": [
      {
        "name": "onsetSymptoms",
        "date": {
          "range": {
            "min": {
              "$date": "2020-03-19T00:00:00Z"
            },
            "max": {
              "$date": "2020-03-19T00:00:00Z"
            }
          }
        }
      },
      {
        "name": "confirmed",
        "date": {
          "range": {
            "min": {
              "$date": "2020-03-25T00:00:00Z"
            },
            "max": {
              "$date": "2020-03-25T00:00:00Z"
            }
          }
        }
      }
    ],
    "location": {
      "id": 951.0,
      "country": "Mexico",
      "geometry": {
        "latitude": 20.833333,
        "longitude": -89.0
      }
    },
    "importedCase": {
      "ID": "004-11146",
      "admin_id": 951.0,
      "age": "53",
      "chronic_disease_binary": false,
      "country": "Mexico",
      "date_confirmation": "25.03.2020",
      "date_onset_symptoms": "19.03.2020",
      "geo_resolution": "admin1",
      "latitude": 20.833333,
      "longitude": -89.0,
      "province": "Yucatan",
      "sex": "male",
      "source": "https://www.gob.mx/cms/uploads/attachment/file/543465/Tabla_casos_positivos_COVID-19_resultado_InDRE_2020.03.25.pdf",
      "travel_history_binary": true
    }
  },
```

### CODE REVIEW

File: data-serving/scripts/converters.py [hunk 1]
```diff
@@ -115,3 +116,54 @@ def convert_demographics(id: str, age: str, sex: str) -> Dict[str, Any]:
 
     # If the dictionary is empty, we want to omit it from the JSON output
     return demographics if demographics else None
+
+
+def convert_location(id: str, location_id: float, country: str, adminL1: str, adminL2: str, locality: str, latitude: float, longitude: float) -> Dict[str, Any]:
```
Comment (axmb) [inline data-serving/scripts/converters.py:121]: Can we linebreak this signature?

File: data-serving/scripts/converters.py [hunk 2]
```diff
@@ -115,3 +116,54 @@ def convert_demographics(id: str, age: str, sex: str) -> Dict[str, Any]:
...
+
+    if pd.notna(adminL2):
+        location['administrativeAreaLevel2'] = adminL2
+
+    if pd.notna(locality):
+        location['locality'] = locality
+
+    geometry = {}
+
+    try:
```
Comment (axmb) [inline data-serving/scripts/converters.py:146]: Abstracting some of these conversion subroutines (e.g. `convertIfNotNa`, `convertIfNotNaFloat`) is something to consider as we continue to add conversion logic.

File: data-serving/scripts/converters.py (hunk 1 above)
Comment (khmoran) [reply data-serving/scripts/converters.py]: got this largely to work automatically with

```
  "python.linting.pep8Args": [
    "--max-line-length=100"
  ],
  "python.formatting.autopep8Args": [
    "--max-line-length", "100", "--experimental"
  ],

```
though it still can't break long comments -- but i got a sweet vertical ruler in vscode: `"editor.rulers": [100]` -- so I can see when it's a problem!

File: data-serving/scripts/converters.py (hunk 2 above)
Comment (khmoran) [reply data-serving/scripts/converters.py:149]: fair, I'll think about it with the next PR! thanks!

**Comment by khmoran** [comment]
[Ingest existing line-list data into MongoDB](https://trello.com/c/GzlFTuYe/83-ingest-existing-line-list-data-into-mongodb)

**Pull request merged**